MekHQ/src/mekhq/gui/TOETab.java

```diff
@@ -96,12 +96,13 @@ public void initTab() {
         orgTree.setDropMode(DropMode.ON);
         orgTree.setTransferHandler(new TOETransferHandler(getCampaignGui()));
 
-        scrollForceView = new JScrollPane();
-        scrollForceView.setMinimumSize(new java.awt.Dimension(550, 600));
-        scrollForceView.setPreferredSize(new java.awt.Dimension(550, 600));
-        scrollForceView.setHorizontalScrollBarPolicy(ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
+        panForceView = new JPanel();
+        panForceView.setMinimumSize(new java.awt.Dimension(550, 600));
+        panForceView.setPreferredSize(new java.awt.Dimension(550, 600));
+        panForceView.setLayout(new BorderLayout());
+        //panForceView.setHorizontalScrollBarPolicy(ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
```

Comment:
Can this be removed, or is this to remind folks not to do this?

Reply:
It can removed and I have now done so in commit 9a200a7b1. It isn't even a method for panels. It was a relic of that formerly being a scroll pane.